smdebug/tensorflow/version_check.py

```diff
@@ -1,11 +1,11 @@
 # Third Party
-import tensorflow as tf
+import tensorflow.compat.v1 as tf
 from packaging import version
 
 # Separating this out to a different file because black doesn't like any code before imports
 # So this can't go at the top of init.py
 # If this is not at the top, then we see unhelpful messages as we fail to import some modules
-if version.parse(tf.__version__) >= version.parse("2.0.0") or version.parse(
-    tf.__version__
-) < version.parse("1.13.0"):
-    raise ImportError("SMDebug only supports TensorFlow 1.13.x, 1.14.x and 1.15.x")
+if version.parse(tf.__version__) < version.parse("1.13.0"):
+    raise ImportError(
+        "SMDebug only supports TensorFlow 1.13, 1.14, 1.15, and experimental functionality in 2.x."
```

Comment:
Didn't you say we are dropping 1.13. I believe the library would crash with 1.13. Please change the message and version number being checked